website/docs/reference/widgets/menu-button.md

```diff
@@ -1,76 +1,107 @@
 # Menu Button
 
-Menu buttons are used to represent a set of actions in a group.
+The Appsmith menu buttons widget is a customizable dropdown menu that allows you to add a list of options for users to select from. It's a powerful tool for creating navigation menus, dropdown lists, and options for users to choose from.
 
-<VideoEmbed host="youtube" videoId="tDMAOxZTmxY" title="How to use Mwnu Button Widget" caption="How to use Mwnu Button Widget"/>
+<VideoEmbed host="youtube" videoId="tDMAOxZTmxY" title="How to use Menu Button Widget" caption="How to use Menu Button Widget"/>
 
-## Properties
+:::info
+The icons are used from [Blueprint](https://blueprintjs.com) library. For more information, see the [list of icons available on Blueprint](https://blueprintjs.com/docs/#icons).
```

Comment:
Sorry! Didn't understand this. Could you please elaborate?

Reply:
I mean that instead of adding that line as a callout, let's add that statement where the red underline is on the screenshot cause it is relevant info there to let the users know which set of icons are available to them.

Reply:
Got it.